syncd/scripts/syncd_init_common.sh

```diff
@@ -233,6 +233,11 @@ config_syncd_mlnx()
     if [[ -f /tmp/sai_extra.profile ]]; then
         cat /tmp/sai_extra.profile >> /tmp/sai.profile
     fi
+
+    if [[ -f /$HWSKU_DIR/independent_mode_support.profile ]]; then
```

Comment:
why need this?

Reply:
got in here by mistake form another commit. removing